### PR TITLE
Adding support for scala 2.12 and spark 2.4

### PR DESCRIPTION
**Pull request by ancasarb**
Should be merged after https://github.com/combust/mleap/pull/493, as the base branch is the one containing the Spark 2.4 changes. Or we could just merge this PR which contains both changes.

Ensures the mleap-benchmark and mleap-executor-tests projects are also included in the build

### CODE REVIEW

**Comment by apurva-sharma** [comment]
@ancasarb , @hollinwilkins  Can I help in getting this merged in anyway?

**Comment by sage3** [comment]
Can I help in moving this PR forward somehow?

**Comment by ceason** [comment]
Any update when this will be merged? We are using databricks which deprecated spark 2.3 some time ago and now only supports spark 2.4.

**Comment by tovbinm** [comment]
Same question here. As we would to like to upgrade to Spark 2.4.3 as well ;0)

**Comment by tovbinm** [comment]
ping @hollinwilkins @ancasarb

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Keys._
 object Dependencies {
   import DependencyHelpers._
 
-  val sparkVersion = "2.3.0"
+  val sparkVersion = "2.4.0"
```
Comment (tovbinm) [inline project/Dependencies.scala]: Better bump to the latest Spark version - `2.4.3`

**Pull request merged**